packages/backend/src/api/endpoints/OrganizationApi.ts

```diff
@@ -116,6 +121,34 @@ export class OrganizationAPI extends AbstractAPI {
     });
   }
 
+  public async updateOrganizationLogo(organizationId: string, params: UpdateLogoParams) {
+    this.requireId(organizationId);
+
+    if (params?.file === null) {
```

Comment:
:upside_down_face: Hmmm.. isn't it a cleaner API if we have two separate methods, one for each endpoint? One uploads a logo and another deletes a logo.

```ts
type UpdateLogoParams = {
  file: Blob | File;
  uploaderUserId: string;
};

updateOrganizationLogo(organizationId: string, params: UpdateLogoParams)
deleteOrganizationLogo(organizationId: string)
```

Reply:
This is indeed a great suggestion. I just followed the Clerkjs implementation, but it makes more sense to separate them.